### ISSUE TITLE
[Feature request] Allow aspect ratio switching for videos that need it

### ISSUE THREAD

**Issue by Pentaphon** [iv-org/invidious#4683]
**Is your feature request related to a problem? Please describe.**
Sometimes, MANY videos on youtube are uploaded in a bad aspect ratio like https://www.youtube.com/channel/UCP0AsEE4luYjzk4NLtNxjpw and it would be handy to be able to view them properly by switching their aspect ratio in the player itself. 

**Describe the solution you'd like**
A button that switches to different aspect ratios to force the video to go 4:3, 16:9, 20:9, etc. 

**Describe alternatives you've considered**
Basically, the aspect ratio switching of VLC is ideal and should be looked at as a model for how players should switch aspect ratio.

**Additional context**
N/A
